src/openmrs-esm-api.ts

```diff
@@ -1,2 +1,6 @@
-export { openmrsFetch } from "./openmrs-fetch";
+export { openmrsFetch, openmrsObservableFetch } from "./openmrs-fetch";
```

Comment:
why the need to implement both promise and observable based fetch?

Reply:
observable api calls can be easily composed together, debounced, and canceled. More so than promises. My thought is that openmrs-esm-api will let the user choose which way they prefer.